app/scripts/views/sign_up.js

```diff
@@ -174,6 +181,16 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
         isEmailOptInVisible: this._isEmailOptInEnabled(),
         isMigration: this.isMigration()
       };
+
+      if (isSync && this.isInExperiment('syncCheckbox')) {
+        this.experimentTrigger('syncCheckbox.triggered');
```

Comment:
The `experimentTrigger` with `<name>.triggered` is repeated in several places, think we should abstract that?